services/secrets/secrets.go

```diff
@@ -1,6 +1,5 @@
 // Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
Do we copyright the year the code was written, or the year it is merged?
I would have thought it is the year it is merged…